text/0000-selecting-default-java-version.md

```diff
@@ -0,0 +1,34 @@
+# Selecting the next default Java Version for Paketo Buildpacks
+
+## Summary
+
+This RFC introduces a process to select the next default version of the Paketo Java Buildpack.
+
+## Motivation
+
+Currently, there is no process how the default version of Java is picked in the Paketo world. As of today (January 2023), the default still is Java 11 (released September 2018) although Java 17 (released September 2021) is already available.
+
+## Detailed Explanation
+
+The default should be changed according to a defined process rather than at some undefined point in time to some undefined version. To do so, the default should be changed once the latest releases LTS version of Java is at least one year old. 
+
+## Rationale and Alternatives
+
+- Do nothing, update default manually from time to time
+- Always use the latest LTS immediately
+  - Could break people immediately
+- This RFC
+  - Gives room to adapt to breaking changes
+
+## Implementation
+
+To fulfill this RFC, the default version should be changed to Java 17. All future versions should be bumped according to this RFC.
```

Comment:
```suggestion
To fulfill this RFC, the default version should be changed to Java 17 immediately. 

Going forward, the default Java version should be bumped according to this RFC.
```

Just trying to make it clear the immediate action and then what we'll do going forward.